pkg/descheduler/strategies/pod_lifetime.go

```diff
@@ -74,7 +74,12 @@ func PodLifeTime(ctx context.Context, client clientset.Interface, strategy api.D
 		excludedNamespaces = strategy.Params.Namespaces.Exclude
 	}
 
-	evictable := podEvictor.Evictable(evictions.WithPriorityThreshold(thresholdPriority))
+	nodeFit := false
+	if strategy.Params != nil {
+		nodeFit = strategy.Params.NodeFit
+	}
+
+	evictable := podEvictor.Evictable(evictions.WithPriorityThreshold(thresholdPriority), evictions.WithNodeFit(nodeFit))
```

Comment:
It's valid for the strategies to schedule the pod to the same node. `evictions.WithNodeFit` does not belong here.

Reply:
Good point, I've removed the feature from the `pod_lifetime` strategy. I'll also update the `README` accordingly.